pkg/apis/kops/networking.go

```diff
@@ -195,80 +195,226 @@ const CiliumIpamEni = "eni"
+	StateDir string `json:"stateDir,omitempty"`
+	// TracePayloadLen has not been implemented in the new templates and may be removed in the future.
+	// Setting this has currently no effect.
+	TracePayloadLen int `json:"tracePayloadlen,omitempty"`
+	// Tunnel specifies the Cilium tunelling mode. Possible values are "vxlan", "geneve", or "disabled".
+	// Default: vxlan
+	Tunnel string `json:"tunnel,omitempty"`
+	// EnableIpv6 enables cluster IPv6 traffic. If both EnableIpv6 and EnableIpv4 are set to false
+	// then IPv4 will be enabled.
+	// Default: false
+	EnableIpv6 bool `json:"enableipv6"`
+	// EnableIpv4 enables cluster IPv4 traffic. If both EnableIpv6 and EnableIpv4 are set to false
+	// then IPv4 will be enabled.
+	// Default: false
+	EnableIpv4 bool `json:"enableipv4"`
+	// MonitorAggregation sets the level of packet monitoring. Possible values are "low", "medium", or "maximum".
+	// Default: medium
+	MonitorAggregation string `json:"monitorAggregation"`
+	// BPFCTGlobalTCPMax is the maximum number of entries in the TCP CT table.
+	// Default: 524288
+	BPFCTGlobalTCPMax int `json:"bpfCTGlobalTCPMax"`
+	// BPFCTGlobalAnyMax is the maximum number of entries in the non-TCP CT table.
+	// Default: 262144
+	BPFCTGlobalAnyMax int `json:"bpfCTGlobalAnyMax"`
+	// PreallocateBPFMaps reduces the per-packet latency at the expense of up-front memory allocation.
+	// Default: true
+	PreallocateBPFMaps bool `json:"preallocateBPFMaps"`
+	// SidecarIstioProxyImage is the regular expression matching compatible Istio sidecar istio-proxy
+	// container image names.
+	// Default: cilium/istio_proxy
 	SidecarIstioProxyImage string `json:"sidecarIstioProxyImage"`
-	ClusterName            string `json:"clusterName"`
-	ToFqdnsEnablePoller    bool   `json:"toFqdnsEnablePoller"`
+	// ClusterName is the name of the cluster. It is only relevant when building a mesh of clusters.
+	ClusterName string `json:"clusterName"`
+	// ToFqdnsEnablePoller determines the implementation of FQDN policies.
+	// if this is set to false, the more powerful DNS proxy-based implementation is used.
+	// Enable this option if you want to use FQDN policies but do not want to use
+	// the DNS proxy. To ease upgrade, users may opt to set this option to "true".
+	// Default: false
+	ToFqdnsEnablePoller bool `json:"toFqdnsEnablePoller"`
+	// ContainerRuntimeLabels determines the container runtime(s) used by Cilium
+	// Supported values are: "none", "containerd", "crio", "docker", "auto"
+	// As of Cilium 1.7.0, Cilium no longer fetches information from the
+	// container runtime and this field is ignored.
+	// Default: none
 	ContainerRuntimeLabels string `json:"containerRuntimeLabels,omitempty"`
-	IPTablesRulesNoinstall bool   `json:"IPTablesRulesNoinstall"`
-	AutoDirectNodeRoutes   bool   `json:"autoDirectNodeRoutes"`
-	EnableNodePort         bool   `json:"enableNodePort"`
-	Ipam                   string `json:"ipam,omitempty"`
+	// Ipam determines the IP address allocation mode to use.
```

Comment:
```suggestion
	// Ipam specifies the IP address allocation mode to use.
```
(I don't think the word "determines" fits well in this context.)